virttest/qemu_storage.py

```diff
@@ -1096,6 +1096,132 @@ def snapshot_apply(self):
 
         process.run(cmd)
 
+    def bitmap_add(self, bitmap_name):
+        """
+        Add a bitmap to a image.
+
+        :note: params should contain:
+               bitmap_name -- the name of bitmap
```

Comment:
:param bitmap_name: The name of the bitmap

remove params line, for we don't need params as an argument here.
Please also update the following functions